cuda_bindings/docs/source/overview.md

```diff
@@ -312,6 +312,22 @@ maximize performance ({numref}`Figure 1`).
 Screenshot of Nsight Compute CLI output of CUDA Python example.
 ```
 
+## Getting the address of underlying C objects from the low-level bindings
+
+Within the low-level object wrappers there are a number of cdef classes which effectively provide a PyObject interface to CUDA types. For example, the CUdevice type is defined by:
```

Comment:
I am a bit hesitant to mention that it is a `cdef class` here. The fact that it's a Cython extension class should be an implementation detail. Can we just say the CUDA types are exposed to Python as Python classes, with certain established methods?